test/Case Insensitive Text.coffee

```diff
@@ -0,0 +1,6 @@
+helpers = require './helpers'
+
+helpers.describe 'Case Insensitive Text', (test) ->
+	string = 'hello world'
+	describe 'validate', ->
+		test.validate(string, true, string)
```

Comment:
I could be wrong but it looks like this function is testing the string against itself?
Would it not be better to compare it against a upper case version of itself?